taxi/urls.py

```diff
@@ -2,11 +2,17 @@
 
 from .views import (
     index,
+    CarCreateView,
+    CarUpdateView,
+    CarDeleteView,
     CarListView,
     CarDetailView,
     DriverListView,
     DriverDetailView,
+    ManufacturerCreateView,
     ManufacturerListView,
+    ManufacturerUpdateView,
+    ManufacturerDeleteView,
 )
 
 urlpatterns = [
```

Comment:
just FYI, u can split urlpatterns into several lists so code can look better and more structured, more about it [here](https://docs.djangoproject.com/en/4.2/topics/http/urls/#including-other-urlconfs)

yr approach is okay until u have a lot of paths